packages/evolution-legacy/src/components/admin/validation/InterviewStats.js

```diff
@@ -45,6 +46,12 @@ class InterviewStats extends React.Component {
     //  parsed: false
     //};
     this.validations = appConfig.getAdminValidations();
+
+    this.KeepDiscardChange = ({ choice, personId }) => {
```

Comment:
Commencer les noms de méthodes par une minuscule ici, donc `this.keepDiscard`

Reply:
Fait.